src/css/showHide.js

```diff
@@ -4,6 +4,23 @@ define( [
 	"../css/var/isHidden"
 ], function( jQuery, dataPriv, isHidden ) {
 
+var defaultDisplayMap = {};
+
+function setDefaultDisplay( elem ) {
+	var doc = elem.ownerDocument,
+		nodeName = elem.nodeName,
+		temp = doc.body.appendChild( doc.createElement( nodeName ) ),
+		display = jQuery.css( temp, "display" );
+
+	temp.parentNode.removeChild( temp );
+
+	if ( display === "none" ) {
+		display = "block";
```

Comment:
Now this is bold move... don't now how i feel about it. 

We definitely need tests from https://github.com/jquery/jquery/commit/60f546acb1c7136092b4fd01cccff052e468cc72